pom.xml

```diff
@@ -191,28 +191,39 @@
 	</build>
 
 	<repositories>
-                <repository>
-                        <id>ulisboa-repo</id>
-                        <url>http://nexjen.ulisboa.pt/nexus/content/groups/public/</url>
-                </repository>
-                <repository>
-                        <id>qubIT-repo-public</id>
-                        <url>https://repository.qub-it.com/nexus/content/groups/public</url>
-                </repository>
-                <repository>
-                        <id>qubIT-repo</id>
-                        <url>https://repository.qub-it.com/nexus/content/groups/qubITRepo</url>
-                </repository>
+		<repository>
+			<id>qubIT-oss</id>
+			<url>https://repository-oss.omnis.cloud/nexus/repository/releases/</url>
+		</repository>
+		<repository>
+				<id>ulisboa-repo</id>
+				<url>http://nexjen.ulisboa.pt/nexus/content/groups/public/</url>
+		</repository>
+		<repository>
+				<id>qubIT-repo-public</id>
+				<url>https://repository.qub-it.com/nexus/content/groups/public</url>
+		</repository>
+		<repository>
+				<id>qubIT-repo</id>
+				<url>https://repository.qub-it.com/nexus/content/groups/qubITRepo</url>
+		</repository>
         </repositories>
 
+	<pluginRepositories>
+		<pluginRepository>
+			<id>qubIT-oss</id>
+			<url>https://repository-oss.omnis.cloud/nexus/repository/releases/</url>
+		</pluginRepository>
+	</pluginRepositories>
+
 	<distributionManagement>
 		<repository>
-			<id>qubIT-repo</id>
-			<url>https://repository.qub-it.com/nexus/content/repositories/releases/</url>
+			<id>qubIT-oss</id>
+			<url>https://repository-oss.omnis.cloud/nexus/repository/releases/</url>
```

Comment:
@joaofreitas-qubit we should publish a new version of this project so it get's into the OSS repository.